### PR TITLE
ipauser, ipahost: Return random password

### PR DESCRIPTION
**Pull request by t-woerner**
ipauser: Return generated random password

The random password is only returned if random is yes and user did not exist or update_password is yes.

ipahost: Return generated random password

The random password is only returned if random is yes and the host did not exist or update_password is yes.

Fixes issue #134 (ipahost does not return the random password)

### CODE REVIEW

**Comment by varunmylaraiah** [comment]
Documentation, READMEs(host and user) and Example playbooks are LGTM

File: playbooks/host/ensure_host_with_randompassword.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+- name: Tests
```
Comment (varunmylaraiah) [inline playbooks/host/ensure_host_with_randompassword.yml]: Playbook to ensure host with a random password

File: playbooks/user/ensure_user_with_randompassword.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+- name: Tests
```
Comment (varunmylaraiah) [inline playbooks/user/ensure_user_with_randompassword.yml]: test Name is missing here too

File: playbooks/host/ensure_host_with_randompassword.yml (hunk 1 above)
Comment (t-woerner) [reply playbooks/host/ensure_host_with_randompassword.yml]: Fixed

File: playbooks/user/ensure_user_with_randompassword.yml (hunk 2 above)
Comment (t-woerner) [reply playbooks/user/ensure_user_with_randompassword.yml]: Fixed

**Pull request merged**